crates/aptos/src/update/mod.rs

```diff
@@ -84,10 +86,19 @@ pub struct UpdateRequiredInfo {
 impl UpdateRequiredInfo {
     pub fn update_required(&self) -> Result<bool> {
         match self.current_version {
-            Some(ref current_version) => bump_is_greater(current_version, &self.target_version)
-                .context(
+            Some(ref current_version) => {
+                // remove ".beta" or ".rc" for version comparison
```

Comment:
Could you document this in the update command? Tell people that if they're trying to update to a new beta or rc version, it won't actually work and they'll need to force it.

Reply:
Actually I'm not sure there is even a way to update in this case, I suppose we just say that the tool doesn't support updating to RC / beta versions.

Reply:
movefmt is currently under beta so the version contains "beta" to mark it: `https://github.com/movebit/movefmt/releases/tag/v1.0.0.beta`. Do you think we should instead remove it from tag?

Reply:
I think tagging things with beta or rc makes sense, it's just that this code ignores it. Is it intentional that we provide no way to upgrade if these tags are present? e.g. as it is now, the tool will assume it's up to date if the user has 1.0.0.rc1 even if 1.0.0.rc2 is available. I think this is fine actually, just might be good to leave a comment explaining this behavior.

Reply:
Added a comment as you suggested